qiling/loader/pe.py

```diff
@@ -15,6 +15,35 @@
 from .loader import QlLoader
 from qiling.os.memory import QlMemoryHeap
 
+
+class QlPeCacheEntry:
+    def __init__(self, data, cmdlines, import_symbols, import_table):
+        self.data = data
+        self.cmdlines = cmdlines
+        self.import_symbols = import_symbols
+        self.import_table = import_table
+
+# A default simple cache implementation
+class QlPeCache:
+    def create_filename(self, path, address):
+        return path + ".%x.cache2" % address
```

Comment:
Since the file format is changed slightly, I made sure that the names are different so it will not try to load old cache files.